Tools/CMake/AMReXOptions.cmake

```diff
@@ -180,6 +180,12 @@ if (AMReX_CUDA OR AMReX_HIP)
    print_option(GPUS_PER_NODE)
 endif ()
 
+#
+# GPU RDC support
+#
+cmake_dependent_option( AMReX_GPU_RDC "Enable GPU RDC" OFF
```

Comment:
In order to play well with community standards in dependent projects, let's add this logic: https://cmake.org/cmake/help/latest/prop_tgt/CUDA_SEPARABLE_COMPILATION.html#prop_tgt:CUDA_SEPARABLE_COMPILATION
```suggestion
# https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_SEPARABLE_COMPILATION.html
set(_GPU_RDC_default OFF)
if(DEFINED CMAKE_CUDA_SEPARABLE_COMPILATION)
    set(_GPU_RDC_default "${CMAKE_CUDA_SEPARABLE_COMPILATION}")
endif()
cmake_dependent_option( AMReX_GPU_RDC "Enable GPU RDC" ${_GPU_RDC_default}
unset(_GPU_RDC_default)
```